omnibus/Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: f64a223f8a90a8fe74154e155564ca9211de0c38
+  revision: aa90acf328a6d23125ec498de7a50b6017732f87
```

Comment:
I don't know if this is relevant but the updated Gemfile.lock here is likely to cause you merge conflicts.